### DISCUSSION TITLE
print() to a file is senstive to shell terminal carriage size

### DISCUSSION THREAD

**Discussion by scottb5612** [Textualize/rich#3570] [Q&A]
I am  writing long lines to a file using print(),
which is intercepted by rich.

When I print to a file,  extra line feeds
are added to enforce a carriage size which is sensitive to 
that of the shell window from which I invoked the program.
Perhaps there is a hidden variable  that I need to set via
an appropriate call?

Here is a minimal example

from rich import print
long_string = 'string_hippopotomonstrosesquipédaliophobie_!@#$%^&*()_+Supercalifragilisticexpialidocious_Antidisestablishmentarianism1234567890123456789'
with open("test.csv", "w") as f:
    print(long_string, file=f)

For example, I make the window suitably narrow, I get 3 lines of output, when I expect only 1:

string_hippopotomonstrosesquipédaliophobie_!@#
$%^&*()_+Supercalifragilisticexpialidocious_An
tidisestablishmentarianism1234567890123456789

**Comment by willmcgugan** [comment]
Rich print will perform word wrapping. If you don't want word wrapping, because you're writing to a file, then consider not overriding print for that.

**Comment by scottb5612** [comment]
Is there f Be grain  control, that is, can I disable it for period of time during the running of the program does it have to be one way the other during the whole program run??
